### PR TITLE
[4] Empty State for Banner clients and tracks

### PR DESCRIPTION
**Pull request by PhilETaylor**
replacement for closed #33376 with changes

### Summary of Changes

see #33376

+ Hides buttons in line with empty state design principle
+ slight code cleanup

### Testing Instructions

see #33376

### Actual result BEFORE applying this Pull Request

No empty state 

### Expected result AFTER applying this Pull Request

Empty state designed

Screenshot 2021-04-27 at 20 13 50
Screenshot 2021-04-27 at 20 13 42

### CODE REVIEW

File: administrator/components/com_banners/tmpl/tracks/emptystate.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Layout\LayoutHelper;
+
+$displayData = [
+	'textPrefix' => 'COM_BANNERS_TRACKS',
+	'formURL'    => 'index.php?option=com_banners&view=tracks',
```
Comment (Quy) [inline administrator/components/com_banners/tmpl/tracks/emptystate.php]:
```suggestion
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b4404b78ce2d94e8316145a330fdeb2d391f163c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33381">issues.joomla.org/tracker/joomla-cms/33381</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on b4404b78ce2d94e8316145a330fdeb2d391f163c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33381">issues.joomla.org/tracker/joomla-cms/33381</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33381">issues.joomla.org/tracker/joomla-cms/33381</a>.</sub>

**Comment by drmenzelit** [comment]
Thanks

**Pull request merged**